lib/network.js

```diff
@@ -1,22 +1,27 @@
 var assert = require('assert')
-var swarmDefaults = require('dat-swarm-defaults')
-var disc = require('discovery-swarm')
+var pump = require('pump')
+var hyperswarm = require('hyperswarm')
 
 module.exports = function (archive, opts, cb) {
   assert.ok(archive, 'dat-node: lib/network archive required')
   assert.ok(opts, 'dat-node: lib/network opts required')
 
   var DEFAULT_PORT = 3282
```

Comment:
hyperswarm is not compatible to discovery-swarm. Is it a good idea to use the same default port?

Reply:
If we use a different port then people may have to figure out how to re-open new ports on their firewalls

Reply:
🤔 - If a current dat-node is registered _somewhere_ in the discovery-swarm and the node is updated to hyperswarm then a discovery-swarm version might connect (eventually consistent) to the hyperswarm dht. Not sure what happens in that case.

Reply:
Do you have a reproducible test case? It seems like the connection would be reset once the discovery-swarm instance goes down. The peers won't be able to find eachother on the second pass

Reply:
It wouldn't connect to the DHT since that's on a different port, the hardcoded port is for the TCP server, if a discovery-swarm connects to that it'll be fine since it's using the same version of hyperdrive